src/encoded/static/components/dbxref.js

```diff
@@ -107,6 +107,9 @@ export const dbxrefPrefixMap = {
     Cellosaurus: {
         pattern: 'https://web.expasy.org/cellosaurus/{0}',
     },
+    FactorBook: {
+        pattern: 'http://factorbook.beta.wenglab.org/experiment/{0}',
+    },
```

Comment:
would like to double-check with DAC is the URL not going to change to "non-beta" soon

Reply:
this page is SUPER slow